internal/service/project/resource_project_test.go

```diff
@@ -980,12 +981,106 @@ func TestAccProject_withInvalidLimitNameOnUpdate(t *testing.T) {
 	})
 }
 
+func TestAccProject_withTags(t *testing.T) {
+	var (
+		orgID        = os.Getenv("MONGODB_ATLAS_ORG_ID")
+		projectName  = acc.RandomProjectName()
+		nameUpdated  = "my-tag-name-updated"
+		envUnchanged = "unchanged"
+		tagsEmpty    = map[string]string{}
+		tags1        = map[string]string{
+			"Name":        "my-tag-name",
+			"Environment": envUnchanged,
+			"Deleted":     "short-lived",
+		}
+		tagsOneUpdatedOneDeleted = map[string]string{
+			"Name":        nameUpdated,
+			"Environment": envUnchanged,
+			"NewKey":      "new-value",
+		}
+		tagsOnlyIgnored = map[string]string{
+			"Name": nameUpdated,
+		}
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:                 func() { acc.PreCheckBasic(t) },
+		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
+		CheckDestroy:             acc.CheckDestroyProject,
+		Steps: []resource.TestStep{
+			// no tags
```

Comment:
knit: have comments at the right of Config lines (if needed)

Reply:
Agree. They don't add much value. Removed in c8208d9